MMM-MBTA.js

```diff
@@ -171,24 +253,51 @@ Module.register("MMM-MBTA", {
                             seconds = "0" + seconds;
                         }
                         
-                        preAwayCell.innerHTML = minutes + ":" + seconds;
+                        preETACell.innerHTML = minutes + ":" + seconds;
                     } else {
-                        preAwayCell.innerHTML = seconds;
+                        preETACell.innerHTML = seconds;
                     }
                 }
-                row.appendChild(preAwayCell);
+                row.appendChild(preETACell);
             }
             
+            // Arrival time
             if (this.config.showArrivalTime) {
-                // Arrival time
                 var arrTimeCell = document.createElement("td");
                 if (config.timeFormat === 24) {
-                    arrTimeCell.innerHTML = moment(parseInt(this.stationData[i].preDt) * 1000).format("H:mm");
+                    if (isNaN(this.stationData[i].preArr) === true) {
+                        arrTimeCell.innerHTML = "No arrival";
+                    } else {
+                        arrTimeCell.innerHTML = moment.unix(this.stationData[i].preArr).format("H:mm");
+                    }
                 } else {
-                    arrTimeCell.innerHTML = moment(parseInt(this.stationData[i].preDt) * 1000).format("h:mm");
+                    if (isNaN(this.stationData[i].preArr) === true) {
+                        arrTimeCell.innerHTML = "No arrival";
+                    } else {
+                        arrTimeCell.innerHTML = moment.unix(this.stationData[i].preArr).format("h:mm");
+                    }
```

Comment:
Good point. Changed.

Sometimes `arrival_time` is `null`. If it's a bus/subway, I'm not sure why. But if it's a train, it's typically because it already arrived and is waiting to depart.

Reply:
Oh god, we need to check for `null`?

If that's the case, we can check nullity just by doing the following:
```js
if (!this.stationData[i].preArr) {
    ...
}
```
This is because `null` is a falsey value, so when you negative null, it returns true.

Thanks javascript.

Reply:
Ahh, right, one of my favorite things about JS...